### PR TITLE
remove string from log

### PR DESCRIPTION
**Pull request by jburel**
Remove string from log
see https://github.com/ome/omero-blitz/pull/93 for testing instructions
cc @dominikl @mtbc

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-insight-push#449](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-insight-push/449/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-insight-push/449/consoleText) for more details.
Possible conflicts:
  - PR #123 jburel 'bump packager version'
    - build.gradle
  - PR #124 jburel 'Add support for third party jars'
    - build.gradle

--conflicts

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-insight-push#449](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-insight-push/449/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-insight-push/449/consoleText) for more details.
Possible conflicts:
  - Upstream changes
    - build.gradle

--conflicts

File: build.gradle [hunk 1]
```diff
@@ -46,7 +46,10 @@ dependencies {
     implementation("org.openmicroscopy:omero-gateway:5.6.3") {
         // Conflicts with `net.java.dev.jna`
         exclude group: "com.sun.jna", module: "jna"
+        // Exclude Bio-Formats api
```
Comment (joshmoore) [inline build.gradle:49]: Maybe say why since that much is obvious from the next line.

Comment (joshmoore) [reply build.gradle]: Thanks.

**Comment by dominikl** [comment]
Looks good. All mentions of passwords replaced by `***` 👍

**Pull request merged**